src/map/status.cpp

```diff
@@ -8357,6 +8474,16 @@ static signed short status_calc_smatk(struct block_list *bl, struct status_chang
 		smatk += sc->data[SC_JAWAII_SERENADE]->val2;
 	if (sc->data[SC_SPELL_ENCHANTING])
 		smatk += sc->data[SC_SPELL_ENCHANTING]->val2;
+	if (sc->data[SC_TALISMAN_OF_MAGICIAN])
+		smatk += sc->data[SC_TALISMAN_OF_MAGICIAN]->val2;
+	if (sc->data[SC_T_FIFTH_GOD])
+		smatk += sc->data[SC_T_FIFTH_GOD]->val2;
+	if (sc->data[SC_ATTACK_STANCE])
+		smatk += sc->data[SC_ATTACK_STANCE]->val3;
+	if (sc->data[SC_TEMPORARY_COMMUNION])
+		smatk += sc->data[SC_TEMPORARY_COMMUNION]->val2;
```

Comment:
I'm testing the skill Temporary Communion on rA master and this branch.
With skill level 5 `sc->data[SC_TEMPORARY_COMMUNION]->val2;` return 50. That is incorrect, should be 15 (skill_lv * 3).
Same thing with `patk` and `hplus`.

![meo](https://user-images.githubusercontent.com/9859310/194995327-f56f2f1d-a7c9-4a39-a3d6-1e43dd6a4a1f.gif)